batch7crud-roll8/src/main/java/com/lftechnology/batch7crud/services/EmployeeService.java

```diff
@@ -2,19 +2,34 @@
 
 import com.lftechnology.batch7crud.dao.EmployeeDao;
 import com.lftechnology.batch7crud.exception.DataException;
+import com.lftechnology.batch7crud.exception.ValidationException;
 import com.lftechnology.batch7crud.model.Employee;
+import com.lftechnology.batch7crud.validator.EmployeeValidator;
 
 import java.util.List;
+import java.util.Map;
 
 /**
  * Created by Grishma Shrestha grishmashrestha@lftechnology.com on 1/19/16.
  */
 
 public class EmployeeService {
-  private EmployeeDao employeeDao = new EmployeeDao();
+  private EmployeeDao employeeDao;
+  private EmployeeValidator validator;
 
-  public Employee create(Employee employee) throws DataException {
-    return employeeDao.create(employee);
+  public EmployeeService() {
+    employeeDao = new EmployeeDao();
+    validator = new EmployeeValidator();
+  }
+
+  public Employee create(Employee employee) throws DataException, ValidationException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: !unknownSymbol!, !unknownSymbol! [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1160)